plutus-benchmark/marlowe/test/role-payout/9.6/057ebc80922f16a5f4bf13e985bf586b8cff37a2f6fe0f3ce842178c16981027.eval.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 194025187
-| mem: 663336})
\ No newline at end of file
+({cpu: 195267187
```

Comment:
Here we go one bazillion golden files. This would be so much easier to review if the diff was accompanied by a graph clearly showing the difference for all the data points at once.

Reply:
> Here we go one bazillion golden files. This would be so much easier to review if the diff was accompanied by a graph clearly showing the difference for all the data points at once.

It would help a lot if we even had all the results printed in a table in a single file.  That should be doable for the marlowe files, although I'm not sure if/how you'd be able to do it more generally.